runtime/ffi.h

```diff
@@ -279,7 +279,7 @@ inline int64_t ffi_c2php(int64_t v) { return v; }
 inline double ffi_c2php(float v) { return v; }
 inline double ffi_c2php(double v) { return v; }
 inline string ffi_c2php(const char *v) { return string(v); }
```

Comment:
This is unchanged, but below is changed. It is ok?

Reply:
Checked again, all right.

Reply:
Discussed offline. Such change can't be done because `string(const char *)` constructor marked `explicit`